### PR TITLE
Test that the distribution tarball is complete

### PR DESCRIPTION
**Pull request by rhansen**

### CODE REVIEW

**Comment by rhansen** [comment]
Set to draft until PR #222 is merged.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -39,6 +41,60 @@ jobs:
...
+          git archive -o "${tmpdir}"/git-repo.tar.gz HEAD
+          mkdir "${tmpdir}"/git-repo
+          cd "${tmpdir}"/git-repo
+          tar xvfa ../git-repo.tar.gz
+          # Delete files checked into Git that shouldn't be in the
+          # distribution tarball.
+          rm -rf \
+              .github \
+              .gitignore \
+              docs/ipv6-design-doc.md \
```
Comment (SuperSandro2000) [inline .github/workflows/ci.yml:65]:
```suggestion
              docs/ \
```

Comment (rhansen) [reply .github/workflows/ci.yml]: We might add docs here that we do want included in the tarball.

**Comment by rhansen** [comment]
> We should think about putting it into some sort of CI script

Done.

File: .github/workflows/scripts/dist-tarball-check [hunk 2]
```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
```
Comment (SuperSandro2000) [inline .github/workflows/scripts/dist-tarball-check:1]: This file should have a .sh ending.

Comment (rhansen) [reply .github/workflows/scripts/dist-tarball-check:1]: I don't like putting suffixes on executables in case the script is rewritten in another language.

**Pull request merged**